<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimationOptionsGroupBox</class>
 <widget class="QGroupBox" name="AnimationOptionsGroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>108</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Animation configuration optionss</string>
  </property>
  <property name="title">
   <string>Animation configuration options</string>
  </property>
  <property name="flat">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_10">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="speedL">
         <property name="text">
          <string>Speed</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="speedSB">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="delayLabel">
         <property name="text">
          <string>Delay</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="delaySB">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="iterationsL">
         <property name="text">
          <string>Iterations</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="iterationsSB">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="directionL">
         <property name="text">
          <string>Direction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="directionCB">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>Backward</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Forward</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="achsesL">
         <property name="text">
          <string>Achses</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="achsesCB">
         <item>
          <property name="text">
           <string>X-Axis</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y-Axis</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Z-Axis</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="textAnimationL">
         <property name="text">
          <string>Text</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textAnimationLE">
         <property name="maxLength">
          <number>50</number>
         </property>
         <property name="placeholderText">
          <string>Text to display</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="fireworkParticlL">
         <property name="text">
          <string>Firework particls</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="fireworkParticlSB"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="ledsL">
         <property name="text">
          <string>LEDs</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="ledsSB">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>512</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QLabel" name="invertL">
         <property name="text">
          <string>Invert</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="invertSB">
         <property name="toolTip">
          <string>Um zu invertieren setze den Wert auf 1</string>
         </property>
         <property name="maximum">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyPB">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
